tests/drivers/mspi/api/boards/nrf54l15dk_nrf54l15_cpuapp.overlay

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	aliases {
+		mspi0 = &sdp_mspi;
+	};
+};
+
+&cpuflpr_vpr {
+	status = "okay";
+
+	sdp_mspi: sdp_mspi@5004c000 {
+		mspi_device: mspi_device@0 {
+			status = "okay";
+			compatible = "zephyr,mspi-emul-device";
+			reg = <0x0>;
+			mspi-max-frequency = <48000000>;
+		};
+	};
```

Comment:
As mentioned offline, this redefinition is needed because those lines:
https://github.com/nrfconnect/sdk-nrf/pull/18893/files#diff-46fdf8d2608e604b87bd6b25d0cb3243ecc117f0d63f536def7c8e8532a09c41R87-R94
are applied only after this file.
I will look into how this could be solved, until now it would be nice to add a comment here about the issue.

Reply:
I've modified the driver a bit so that it's not necessary to use the address. Unfortunately, the order is still important.